patches/chrome-utility-BUILD.gn.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
+old mode 100644
+new mode 100755
+index 854e99acdf1cc6a682ff13feb387d198efc06273..1595edd711b7d57dac537a2714e2b943304027a0
+--- a/chrome/utility/BUILD.gn
++++ b/chrome/utility/BUILD.gn
+@@ -25,6 +25,7 @@ static_library("utility") {
+ 
+   public_deps = []
+   deps = [
++    "//brave/utility",
```

Comment:
This doesn't cause circular dependency?
`//brave/utility` target has `//chrome/utility` in its dep list.

Reply:
nope